docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1097,17 +1079,17 @@ fail! Here's a few:
 
 <strong>Core Concept</strong>
 
-With Cypress, you don't have to assert to have a useful test. Even without
-assertions, a few lines of Cypress can ensure thousands of lines of code are
-working properly across the client and server!
+With Cypress, you don't have to write explicit assertions to have a useful test.
+Without a single `expect()` or `.should()`, a few lines of Cypress can ensure
+thousands of lines of code are working properly across the client and server.
 
 This is because many commands have a built in
-[Default Assertion](#Default-Assertions) which offer you a high level of
-guarantee.
+[Implicit Assertions](#Implicit-Assertions) which offer you a high level of
+confidence that your application is working as expected.
 
 :::
 
-### Default Assertions
+### Implicit Assertions
 
 Many commands have default, built-in assertions, or rather have requirements
 that may cause it to fail without needing an explicit assertion you've added.
```

Comment:
should we mention each command will outline these in their docs?

Reply:
I'm not sure that needs to be said explicitly - just feels like words for the sake of words.

Would you as a new user *not* expect each command page's to explain it?